amazon-kinesis-client/src/main/java/software/amazon/kinesis/lifecycle/ShutdownTask.java

```diff
@@ -180,6 +192,19 @@ private void createLeasesForChildShardsIfNotExist()
         }
     }
 
+    private void updateLeasesForChildShards()
+            throws DependencyException, InvalidStateException, ProvisionedThroughputException {
+        final Lease currentLease = leaseCoordinator.getCurrentlyHeldLease(shardInfoIdProvider.apply(shardInfo));
+        Set<String> childShardIds = new HashSet<>();
+        for (ChildShard childShard : childShards) {
+            childShardIds.add(childShard.shardId());
+        }
```

Comment:
nit: We can also use `childShards.stream().map(ChildShard::shardId).collect(Collectors.toSet())` for readability